src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/AttachedCollections/PackageContentFileItem.cs

```diff
@@ -0,0 +1,73 @@
+    {
+        private readonly IFileIconProvider _fileIconProvider;
+        private readonly PackageContentFilesGroupItem _groupItem;
+        private readonly AssetsFileTargetLibraryContentFile _contentFile;
+
+        public PackageContentFileItem(IFileIconProvider fileIconProvider, PackageContentFilesGroupItem groupItem, AssetsFileTargetLibraryContentFile contentFile)
+            : base(contentFile.Path)
+        {
+            _fileIconProvider = fileIconProvider;
+            _groupItem = groupItem;
+            _contentFile = contentFile;
+        }
+
+        // All siblings are content files, so no prioritization needed (sort alphabetically)
+        public override int Priority => 0;
+
+        public override ImageMoniker IconMoniker => _fileIconProvider.GetFileExtensionImageMoniker(_contentFile.Path);
+
+        public override object? GetBrowseObject() => new BrowseObject(this);
+
+        public IAttachedCollectionSource? ContainedByAttachedCollectionSource => new MaterializedAttachedCollectionSource(this, new[] { _groupItem });
+
+        private sealed class BrowseObject : BrowseObjectBase
+        {
+            private readonly PackageContentFileItem _item;
+
+            public BrowseObject(PackageContentFileItem item) => _item = item;
+
+            public override string GetComponentName() => _item.Text;
+
+            public override string GetClassName() => VSResources.PackageContentFileBrowseObjectClassName;
+
+            [BrowseObjectDisplayName(nameof(VSResources.PackageContentFilePathDisplayName))]
+            [BrowseObjectDescription(nameof(VSResources.PackageContentFilePathDescription))]
+            public string Path => _item._contentFile.Path;
+
+            [BrowseObjectDisplayName(nameof(VSResources.PackageContentFileOutputPathDisplayName))]
+            [BrowseObjectDescription(nameof(VSResources.PackageContentFileOutputPathDescription))]
+            public string? OutputPath => _item._contentFile.OutputPath;
+
+            [BrowseObjectDisplayName(nameof(VSResources.PackageContentFilePPOutputPathDisplayName))]
+            [BrowseObjectDescription(nameof(VSResources.PackageContentFilePPOutputPathDescription))]
+            public string? PPOutputPath => _item._contentFile.PPOutputPath;
+
+            [BrowseObjectDisplayName(nameof(VSResources.PackageContentFileCodeLanguageDisplayName))]
+            [BrowseObjectDescription(nameof(VSResources.PackageContentFileCodeLanguageDescription))]
+            public string? CodeLanguage => _item._contentFile.CodeLanguage;
+
+            [BrowseObjectDisplayName(nameof(VSResources.PackageContentFileBuildActionDisplayName))]
+            [BrowseObjectDescription(nameof(VSResources.PackageContentFileBuildActionDescription))]
+            public string? BuildAction => _item._contentFile.BuildAction;
+
+            [BrowseObjectDisplayName(nameof(VSResources.PackageContentFilePathDisplayName))]
```

Comment:
Wrong resource.. should be Copy To Output

Reply:
Thanks!